firstvoices/backend/tasks/visibility_tasks.py

```diff
@@ -82,7 +82,7 @@ def bulk_change_visibility(job_instance_id):
     # Resume search indexing for site, + reindex entire site
     indexing_paused_feature[0].is_enabled = False
     indexing_paused_feature[0].save()
-    sync_all_site_content_in_indexes(site.id)
+    request_sync_all_site_content_in_indexes(site)
```

Comment:
Is there any benefit to queuing this as a separate task? This is already running in the background

Reply:
I do think we could probably save the job before running the reindex

Reply:
We chatted in person. +1 for queuing this separately so we can track the task separately in the logs.